ample/modelling/rosetta_model.py

```diff
@@ -106,14 +106,14 @@ class RosettaModel(object):
     Class to run Rosetta modelling
     """
 
-    def __init__(self,optd=None,rosetta_dir=None):
-
+    def __init__(self, optd=None, rosetta_dir=None):
+        
         self.debug=None
-
+        
         self.nproc = None
         self.nmodels = None
-        self.run_dir = None # Where the modelling happens
-        self.work_dir = None # Main AMPLE directory
+        self.work_dir = None # Where the modelling happens - can be deleted on exit
+        self.ample_dir = None # Main AMPLE directory
```

Comment:
Some of these comments could be deleted, Felix has been preaching about removing comments and replacing them with descriptive variable names. Here ample_dir is sufficiently descriptive imo to get that it is the main ample directory.